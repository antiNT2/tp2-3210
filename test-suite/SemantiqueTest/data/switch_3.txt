num a;
enum A {
    B, C, D
}
A var;

switch(var){
    case B:
        a = 0;
        break;
    case 2:
        a = 1;
        break;
}
