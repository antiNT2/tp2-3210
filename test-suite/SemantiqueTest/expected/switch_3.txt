Invalid type in case of integer 2